src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -220,6 +223,7 @@ const TradeRoutes = memo(({ isCompact, isStandalone, onChangeTab }: TradeRoutesP
             path={TradeRoutePaths.VerifyAddresses}
             element={verifyAddresses}
           />
+          <Route key={TradeRoutePaths.Sweep} path={TradeRoutePaths.Sweep} element={multiHopSweep} />
```

Comment:
Doing like Ledger verify addresses, pre-sweep step in case the user doesn't have any UTXO honoring the amount!

Reply:
We don't need that anymore considering we don't sweep anymore but use multiple UTXOs in [8aad17b](https://github.com/shapeshift/web/pull/9222/commits/8aad17b5bc9be5a528d329651d22fa59044b852e)